docs/content/ingestion/batch-delta-ingestion.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: doc_page
+---
+
```

Comment:
Also I think this is a nice place to mention that updates can be an expensive process in Druid depending on what is done